wrapper/sessions_test.go

```diff
@@ -0,0 +1,57 @@
+package wrapper
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	_ "github.com/joho/godotenv/autoload"
+)
+
+func TestDataRace(t *testing.T) {
```

Comment:
comment on `TestConnect` that tests the connect to websocket funtionality and also heartbeat, listen functions.